lesson07/index.js

```diff
@@ -0,0 +1,45 @@
+const imageArray = [
+    {
+        img: "1.png", 
+        alt:"画像1", 
+        text: "ブックマーク"
+    }, 
+    {
+        img: "2.png", 
+        alt:"画像2", 
+        text: "メッセージ"
+    }
+];
+
+const ul = document.getElementById('js-list');
+const loading = document.getElementById('js-loading');
+const fragment = document.createDocumentFragment();
+
+const createLoading = () => {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loading.appendChild(loadingImg);
+};
+
+
+new Promise((resolve) => {
+    createLoading();
+    setTimeout(() => {
+        resolve(imageArray);
+    }, 3000);
```

Comment:
I will share what I have learned from other students :)!

createLoading() should not be included in Promise.
The Promise process is supposed to be a server-side process.
Therefore, it should not be written here.

the role of this code is to retrieve data.
How about including Promise in one function?
ex.) function fetchData()...

Reply:
Thank you for review!
Add fetchData function moved promise object as well.
[a674d45](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/7/commits/a674d45005053a8691b1595567f05ec53251ee93)